File: 02-Tech-Module/02-Software-Technologies/_exam/SoftUni_Labs_Solutions/JavaScript/softunilabs/models/lab.js
const Sequelize = require('sequelize');

module.exports = function (sequelize) {
    let Lab = sequelize.define("Lab", {
        name: {
            type: Sequelize.TEXT,
            allowNull: false
        },
        capacity: {
            type: Sequelize.INTEGER,
            allowNull: false
        },
        status: {
            type: Sequelize.TEXT,
            allowNull: false
        }
    }, {
        timestamps: false
    });

    return Lab;
};

File: 02-Tech-Module/02-Software-Technologies/_exam/SoftUni_Labs_Solutions/CSharp/SoftUniLabs/Models/Lab.cs
using System.ComponentModel.DataAnnotations;

namespace SoftUniLabs.Models
{
    public class Lab
    {
		[Key]
		public int Id { get; set; }

		[Required]
		public string Name { get; set; }

		[Required]
		public int Capacity { get; set; }

		[Required]
		public string Status { get; set; }
	}
}
